addon/appModules/microsoft_msn_weather.py

```diff
@@ -0,0 +1,21 @@
+# WinTenApps/microsoft_msn_weather.py
+# Part of Windows 10 App Essentials collection
+# Copyright 2016 Derek Riemer, released under GPL.
+
+# Provides workarounds for the weather app.
+
+
+import appModuleHandler
+import controlTypes
+import re
+
+#Regexp for deciding whether this ID should be a tab control
+RE_TAB_AUTOMATION_MATCH = re.compile(r"L1NavigationButton_\d+")
+
+class AppModule(appModuleHandler.AppModule):
+	def event_NVDAObject_init(self, obj):
+		if obj.UIAElement.CachedAutomationId == u"SideNavigationBar" and not obj.role in (controlTypes.ROLE_GROUPING, ):
```

Comment:
There are parts of UWP's that are IAccessible, chiefly some top-level elements, so it might be best to make sure we're dealing with UIA. Otherwise, nice work.